### PR TITLE
REACTOR-337 - Updates to default event mapping table don't get deployed

### PR DESCRIPTION
**Pull request by tiyberius**
### Description

- [ ] Existing GitHub Issue? Please provide #: 

### Acceptance checklist

- [ ] Feedback addressed
- [ ] Tests added or modified appropriately
- [ ] All tests passed

### CODE REVIEW

File: reactor/features/system.py [hunk 1]
```diff
@@ -159,6 +161,35 @@ def setup(new_settings, configuration, namespace, account_id):
...
+    """
+    slack = SlackWeb(configuration.get('slack_webhook_url'))
+
+    simple_query_init_result = reactor.features.simplequery.initialize(account_id, namespace)
+
+    results = {"SimpleQuery": simple_query_init_result}
+
+    final_results = {}
+    for k, v in results.items():
+        if not v:
```
[the remark this answers is not in the document]
Comment (imcgaunn) [reply reactor/features/system.py]: even better:  `final_results[k] = 'Succeeded' if v else 'Failed'`

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{"type": "issue", "check_name": "F821", "description": "undefined name 'init_event_mapping_db'", "content": {"body": "`        init_event_mapping_db()\n`"}, "categories": ["Bug Risk"], "location": {"path": "./reactor/features/system.py", "positions": {"begin": {"line": 131, "column": 9}, "end": {"line": 131, "column": 9}}}, "remediation_points": null}
```
Comment (silvexis) [inline .flake8]: I thought I had placed this file in .gitignore

File: reactor/features/system.py [hunk 3]
```diff
@@ -120,19 +120,21 @@ def setup(new_settings, configuration, namespace, account_id):
...
-        update_reactor_settings(configuration, encrypt_key_arn, **prepared_settings)
         action_msg = "re-initialized"
         response_code = 200
         record_reactor_reinit()
     else:
+        load_event_mappings()
         prepared_settings['reactor_id'] = create_reactor_id()
-        init_event_mapping_db()
-        update_reactor_settings(configuration, encrypt_key_arn, **prepared_settings)
+        load_event_mappings()
```
Comment (silvexis) [inline reactor/features/system.py]: twice?

Comment (tiyberius) [reply reactor/features/system.py]: what do you mean twice? what twice?

**Pull request merged**